package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Various tools for using and integrating with Swagger.",
   "main": "index.js",
   "scripts": {
-    "test": "gulp"
+    "test": "gulp",
+    "test-debug": "node --debug-brk node_modules/.bin/gulp"
```

Comment:
This could be a separate commit, also `"test": "gulp"` could be updated to use `node node_modules/.bin/gulp` if/when you do it.